.vscode/settings.json

```diff
@@ -26,12 +26,14 @@
     "java.format.settings.url": ".vscode/osu-cse-formatter.xml",
     "java.format.settings.profile": "OSU CSE",
     "java.project.sourcePaths": [
+        "doc",
         "src",
-        "test"
+        "test",
+        "bin"
     ],
     "java.project.outputPath": "bin",
     "java.project.referencedLibraries": [
-        "lib/**/*.jar"
+        "c:\\Users\\icema\\OneDrive\\Documents\\GitHub\\PortfolioProject\\lib\\components (1).jar"
```

Comment:
This is probably not what you want. This hardcoded path will not work on other people's machines. I would recommend rolling this back.